### PR TITLE
added multi resume functionality

### PR DESCRIPTION
**Pull request by basulaib**
If a user has 1 resume, the app will automatically parse it. If a user has more, they will be allowed to select which one they want to parse. 👍

### CODE REVIEW

File: src/main.py [hunk 1]
```diff
@@ -1,11 +1,44 @@
 import json
 import requests
+import glob
+import os
```
Comment (chrisngyn) [inline src/main.py:4]: imports should be ordered alphabetically

File: src/main.py [hunk 2]
```diff
@@ -1,11 +1,44 @@
 import json
 import requests
+import glob
+import os
+
+print("Scanning for resume(s)...")
+
+files = [f for f in glob.glob("*.pdf")]
```
Comment (chrisngyn) [inline src/main.py:8]: could use a more concise variable name than `f`

File: src/main.py [hunk 3]
```diff
@@ -1,11 +1,44 @@
...
+            if option in range(0, len(files)):
+                break
+            else:
+                print("Please enter a number between 1 and " + str(len(files)))
+        except ValueError:
+            print("Please enter a number!")
+
+print("Parsing...")
+if len(files) == 1:
+    f = files[0]
```
Comment (chrisngyn) [inline src/main.py:30]: same as prev comment

File: src/main.py [hunk 4]
```diff
@@ -1,11 +1,44 @@
...
-resume = open('./sample.pdf', 'rb')
+resume = open(f, 'rb')
 
-response = requests.post(url, files={ 'resume': resume })
+response = requests.post(url, files={'resume': resume})
 parsed_resume = json.dumps(response.json(), indent=4)
 
-save_response = open('output.txt', 'w')
-save_response.write(parsed_resume)
+f = os.path.splitext(f)[0] + ".txt"  # remove.pdf and add .txt
```
Comment (chrisngyn) [inline src/main.py:40]: remove .pdf? does this delete the .pdf file provided?

**Review by chrisngyn — CHANGES_REQUESTED** [review]
let a couple of comments but I like the changes you're suggesting

**Comment by basulaib** [comment]
Added your suggestions. Hope you like them!

**Review by chrisngyn — APPROVED** [review]
LGTM 👍

**Pull request merged**